Tests/SwiftDocCTests/Infrastructure/DocumentationBundleInfoTests.swift

```diff
@@ -336,7 +336,7 @@ class DocumentationBundleInfoTests: XCTestCase {
                 errorTypeChecking = false
             }
             XCTAssertTrue(errorTypeChecking)
-            XCTAssertEqual(error.localizedDescription, "Unable to decode Info.plist file. Verify that it is correctly formed. Value missing for key inside <dict> at line 24")
+            XCTAssertEqual(error.localizedDescription, "Unable to decode Info.plist file. Verify that it is correctly formed. Value missing for key inside <dict> at line 1")
```

Comment:
This change will break the test as written. We should refactor the test to not require a specific line number, but that shouldn't happen in this PR.

Reply:
Done in: #887